packages/web/src/app/recommend/page.tsx

```
@@ -1,15 +1,51 @@
+  const isSaved = useCallback(
+    (paper: S2Paper) => makeKeys(paper).some((k) => savedKeys.has(k)),
+    [makeKeys, savedKeys],
+  );
+
+  const markSaved = useCallback(
+    (paper: S2Paper) => {
+      const keys = makeKeys(paper);
+      if (keys.length === 0) return;
+      setSavedKeys((prev) => {
+        const next = new Set(prev);
+        keys.forEach((k) => next.add(k));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)